_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 
+import Child from './Child';
 import ErrorableCheckbox from '../form-elements/ErrorableCheckbox';
-import ErrorableSelect from '../form-elements/ErrorableSelect';
-import ErrorableTextInput from '../form-elements/ErrorableTextInput';
-import DateInput from '../form-elements/DateInput';
-import FullName from '../questions/FullName';
-import SocialSecurityNumber from '../questions/SocialSecurityNumber';
-
-import { childRelationships } from '../../utils/options-for-select.js';
-import { isBlank, isValidMonetaryValue } from '../../utils/validations';
-
-// TODO: create unique nodes for each child in applicationData
+import GrowableTable from '../form-elements/GrowableTable.jsx';
 
 class ChildInformationSection extends React.Component {
-  render() {
-    const message = 'Please enter only numbers and a decimal point if necessary (no commas or currency signs)';
+  // TODO(awong): Pull this out into a model.
+  createBlankChild() {
```

Comment:
Do you not need to specify `this.createBlankChild = this.createBlankChild.bind(this)` in the `constructor`?